utils/genexports/genexports.sh

```diff
@@ -30,6 +30,11 @@ for i in $includes; do
 	echo "#include <$i>" >> $outpfile
 done
 
+echo '/* Newlib with GCC 4.7.4 will not export fdopen from stdio.h. */' >> $outpfile
+echo '#if __GNUC__ == 4' >> $outpfile
+echo '	extern FILE* fdopen(int fd, const char* mode);' >> $outpfile
```

Comment:
Rather than doing this, we should be able to get the definition from Newlib's stdio.h by undefining `__STRICT_ANSI__` before we the include block for GCC 4.x...

Also, it's not technically the GCC version doing this -- it's the Newlib version, so technically this is wrong anyway.

Reply:
Yeah, fair. I was just worried about doing that, because I wasn't sure what the repercussions would be in terms of the rest of the headers getting conditionally compiled differently from how they're compiled with the rest of KOS, using `-std=c99`...

...but if there was a problem, it wouldn't even build, so I guess I'd know immediately anyway.